guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -32,7 +32,6 @@ $ hammer activation-key create \
 --name "_My_Activation_Key_" \
 --unlimited-hosts \
 --description "_Example Stack in the Development Environment_" \
---lifecycle-environment "_Development_" \
 --content-view "_Stack_" \
```

Comment:
Don't just delete the --lifecycle-environment line.

1. Delete both the --lifecycle-environment and --content-view lines
2. Instead, add
```
--content-view-environments "_Development_/_Stack_"
```